CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Release Notes
 
+## Version 1.10.0-beta.0 - 2020-04-20
+
+Improvements:
+* faster containerd start by preloading images [#7793](https://github.com/kubernetes/minikube/pull/7793)
+* Add fish completion support [#7777](https://github.com/kubernetes/minikube/pull/7777)
+* Behavior change: start with no arguments uses existing cluster config [#7449](https://github.com/kubernetes/minikube/pull/7449)
+* conformance: add --wait=all, reduce quirks [#7716](https://github.com/kubernetes/minikube/pull/7716)
+* Upgrade minimum supported k8s version to v1.12 [#7723](https://github.com/kubernetes/minikube/pull/7723)
+* Add default CNI network for running wth podman [#7754](https://github.com/kubernetes/minikube/pull/7754)
+* Behavior change: fallback to alternate drivers on failure [#7389](https://github.com/kubernetes/minikube/pull/7389)
+* Add registry addon feature for docker on mac/windows [#7603](https://github.com/kubernetes/minikube/pull/7603)
+* Check node pressure & new option "node_ready" for --wait flag [#7752](https://github.com/kubernetes/minikube/pull/7752)
+* docker driver: Add Service & Tunnel features to windows   [#7739](https://github.com/kubernetes/minikube/pull/7739)
+* Add master node/worker node type to `minikube status` [#7586](https://github.com/kubernetes/minikube/pull/7586)
+* add new wait component apps_running [#7460](https://github.com/kubernetes/minikube/pull/7460)
+* none: Add support for OpenRC init (Google CloudShell) [#7539](https://github.com/kubernetes/minikube/pull/7539)
+* Upgrade falco-probe module to version 0.21.0 [#7436](https://github.com/kubernetes/minikube/pull/7436)
+
+Bug Fixes:
+* Fix multinode cluster creation for VM drivers [#7700](https://github.com/kubernetes/minikube/pull/7700)
+* tunnel: Fix resolver file permissions, add DNS forwarding test [#7753](https://github.com/kubernetes/minikube/pull/7753)
+* unconfine apparmor for kic [#7658](https://github.com/kubernetes/minikube/pull/7658)
+* Fix `minikube delete` output nodename missing with KIC driver [#7553](https://github.com/kubernetes/minikube/pull/7553)
```

Comment:
change kic driver to docker/podman, kic is an internal name for our code base. most users dont know what is kic